install-server.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+
+set -e
+
+readonly ARGS=( "$@" )
+readonly PROGNAME="./install-server.sh"
+
+usage() {
+    cat <<- EOF
+    usage: $PROGNAME options
+
+    Install Trento Server
+
+    OPTIONS:
+       -p --private-key         the private key used by the runner to connect to the hosts
```

Comment:
Maybe we should clarify that we refer to a ssh key and that this key must be authorized in all the trento agents